### PR TITLE
preview page with fixtures

### PR DESCRIPTION
**Pull request by Hemanth-2002**

### CODE REVIEW

File: src/common/components/SkeletonScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Center, Spinner } from '@chakra-ui/react';
+
+export const SkeletonScreen: React.FC = () => {
```
Comment (pbpranavk) [inline src/common/components/SkeletonScreen.tsx:4]: You can use Skeleton component directly from chakra ui, u can check out it's implementation in ExecutionView

File: src/preview/PreviewView.tsx [hunk 2]
```diff
@@ -1,7 +1,56 @@
...
+                        <Sections publicationSections={publicationSections}/>
+                    </GridItem>
+                    <GridItem
+                        css={{
+                            maxHeight: "calc(100vh - 140px)",
+                            overflowY: "auto",
+                        }}
+                    >
+                        {publicationSections.map((publicationSection, index) => (
+                            <RenderSection publicationSection={publicationSection} sectionIndex={String(index)} key={index}/>
```
Comment (pbpranavk) [inline src/preview/PreviewView.tsx:49]: There was another requirement, when user scrolls on this component, we should update the selectedSectionIndex as well so I think you will have to lift the state up and handle this requirement as well.

Comment (Hemanth-2002) [reply src/preview/PreviewView.tsx:49]: That functionality is taken care in selectAndScrollToSection function in Sections component. When user clicks on a particular section the section is highlighted and user is scrolled to that section's content.

File: src/common/components/SkeletonScreen.tsx (hunk 1 above)
Comment (Hemanth-2002) [reply src/common/components/SkeletonScreen.tsx:4]: Modified this component to include skeleton component from chakra UI and reused it in PreviewView and ExecuteView

**Pull request merged**